fili-core/src/test/java/com/yahoo/bard/webservice/models/druid/client/impl/TestDruidWebService.java

```diff
@@ -39,7 +40,7 @@
     );
     private static ObjectWriter writer = mapper.writer();
 
-    public String jsonResponse = "[]";
+    public Producer<String> jsonResponse = () -> "[]";
```

Comment:
Definitely need Changelog entry for this, since it's a breaking change for anyone trying to use this.